### PR TITLE
Docs: Organize variables

### PR DESCRIPTION
**Pull request by oddlittlebird**
This PR organizes variable content a bit.

- Moved the variable types into a table
- Moved some of the variable topics into a subfolder
- Updated links to reflect new locations
- Added aliases to topics that moved

@scoren-gl , something for you to be aware of

### CODE REVIEW

File: docs/sources/menu.yaml [hunk 1]
```diff
@@ -255,28 +255,33 @@
...
-      name: Add data source variable
-    - link: /variables/add-interval-variable/
-      name: Add interval variable
-    - link: /variables/add-ad-hoc-filters/
-      name: Add ad hoc filters
+    - name: Variable types
+      link: /variables/variable-types/
+      children:
+      - link: /variables/variable-types/add-query-variable/
+        name: Add query variable
```
Comment (torkelo) [inline docs/sources/menu.yaml:262]: Do we need Add in the name for every page here? I can get it's ok to have it in the page title for SEO reasons but in the nav it feels a bit superfluous

**Review by torkelo — APPROVED** [review]
Looks good! 

only think the nav menu wording could be simplified

File: docs/sources/menu.yaml (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/menu.yaml:262]: Yes, we do. We don't know which variable a user wants to know about, so we want to make this content as discoverable as possible for the user.

Comment (oddlittlebird) [reply docs/sources/menu.yaml:262]: @scoren-gl will let me know if I am wrong. :)

Comment (oddlittlebird) [reply docs/sources/menu.yaml:262]: Also, the verb in the title signals to the reader that this is a TASK topic with instructions on HOW to do the task, in this case adding the variable.

**Pull request merged**